lib/Elements/Animateds.tsx

```
@@ -0,0 +1,64 @@
+  const tokens = useMemo(() => {
+    if (!text) return [];
+    const splitRegex = /(\s+)/;
+    return text.split(splitRegex).filter((token) => token.length > 0);
+  }, [text]);
+
+  return tokens.map((token, index) => (
+    <AnimatedToken key={`${token}-${index}`} token={token} />
+  ));
+  return tokens.map((token, index) => (
+    <Text key={`${token}-${index}`}>{token}</Text>
+  ));
```

Biome (2.1.2)

[error] This code will never be reached ...

... because this statement will return from the function beforehand


(lint/correctness/noUnreachable)